R/format_ae_specific.R

```diff
@@ -128,8 +123,8 @@ format_ae_specific <- function(outdata,
       stop("Please use extend_ae_specific_duration() to get the calculation of duration!")
     }
 
-    dur <- outdata$dur * NA
-    for (i in 1:ncol(outdata$dur)) {
+    dur <- outdata$dur[, index_total] * NA
+    for (i in 1:ncol(outdata$dur[, index_total])) {
```

Comment:
would `seq(index_total)` be sufficient? same comment for line 142

Reply:
i will investigate. `seq()` would replace the entire `1:ncol(...` sequence. I'll check and do an update.